mos_tests/murano/murano_test.py

```diff
@@ -0,0 +1,134 @@
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+# from unittest import TestCase
+
+import pytest
+from random import randint
+from mos_tests.functions import os_cli
+
+pytestmark = pytest.mark.undestructive
+
+
+def rand_num(a=1, b=1000):
+    return randint(a, b)
+
+
+def user_env(user, password, project):
+    """Export this env if you need to perform actions from another user"""
+    env = 'export OS_TENANT_NAME="{2}" && ' \
+          'export OS_PROJECT_NAME="{2}" && ' \
+          'export OS_USERNAME="{0}" && ' \
+          'export OS_PASSWORD="{1}"'.format(
+            user, password, project)
+    return env
+
+
+def execute_on_all_controllers(env, cmd):
+    controllers = env.get_nodes_by_role('controller')
+    all_output = []
+    for controller in controllers:
+        with controller.ssh() as remote:
+            output = os_cli.CLICLient(remote).execute(cmd)
+            all_output.append(output)
+    return all_output
+
+
+@pytest.mark.xfail(run=False, reason="1551575")
+@pytest.mark.testrail_id('543127')
+def test_user_with_permissions_can_share_pkg(
+        env, openstack_client, cli_to_controller):
+    """QA-1572 [Murano Features] User with all permissions can set
+        'shared' package to 'unshared'
+    https://mirantis.testrail.com/index.php?/cases/view/543127
+    * BUG:
+    *   User with permissions can not set 'unshared' Murano package to 'shared'
+    *   https://bugs.launchpad.net/mos/+bug/1551575
+    Actions:
+    1. Create new role 'can_publicize_packages';
+    2. Create new user inside 'services' project;
+    3. Assign new role to the new user;
+    4. On all controllers add new role to /etc/murano/policy.json:
+        from:
+        "publicize_package": "rule:admin_api",
+        to:
+        "publicize_package": "rule:admin_api or role:can_publicize_packages",
+    5. As a new user import Murano pkg;
+    6. As a new user update imported pkg with Public=True;
+    7. Check that pkg became public.
+    """
+    pkg_for_import = 'io.murano.apps.docker.Interfaces'
+    role_name = 'can_publicize_packages'
+    new_user = {'name': '_test_user_{0}'.format(rand_num()),
+                'password': 'password',
+                'project': 'services'}
+    new_user_env = user_env(new_user['name'], new_user['password'],
+                            new_user['project'])
+
+    # Change OS_AUTH_URL from ":5000/" --> ":5000/v2.0" in /root/openrc
+    cmd = unicode(r"sed -i 's#:5000/\x27#:5000/v2.0\x27#g' /root/openrc")
```

Comment:
I believe that this is product bug. So our tests should test normal behavior, not workaround.